#
# This file is autogenerated by pip-compile
# To update, run:
#
#    pip-compile --output-file=requirements.txt setup.py
#
aiocache==0.11.1          # via synse_server (setup.py)
aiofiles==0.6.0           # via sanic
bison==0.1.3              # via synse_server (setup.py)
cachetools==4.2.1         # via google-auth
certifi==2020.12.5        # via kubernetes, requests
chardet==4.0.0            # via requests
google-auth==1.28.0       # via kubernetes
grpcio==1.36.1            # via synse-grpc, synse_server (setup.py)
httptools==0.1.1          # via sanic
idna==2.10                # via requests
kubernetes==12.0.1        # via synse_server (setup.py)
multidict==5.1.0          # via sanic
oauthlib==3.1.0           # via requests-oauthlib
prometheus-client==0.9.0  # via synse_server (setup.py)
protobuf==3.15.6          # via synse-grpc
pyasn1-modules==0.2.8     # via google-auth
pyasn1==0.4.8             # via pyasn1-modules, rsa
python-dateutil==2.8.1    # via kubernetes
pyyaml==5.4.1             # via bison, kubernetes, synse_server (setup.py)
requests-oauthlib==1.3.0  # via kubernetes
requests==2.25.1          # via kubernetes, requests-oauthlib
rsa==4.7.2                # via google-auth
sanic-routing==0.4.2      # via sanic
sanic==21.3.2             # via synse_server (setup.py)
shortuuid==1.0.1          # via synse_server (setup.py)
six==1.15.0               # via google-auth, grpcio, kubernetes, protobuf, python-dateutil, websocket-client
structlog==21.1.0         # via synse_server (setup.py)
synse-grpc==3.1.0         # via synse_server (setup.py)
ujson==4.0.2              # via sanic
urllib3==1.26.5           # via kubernetes, requests
uvloop==0.15.2            # via sanic
websocket-client==0.58.0  # via kubernetes
websockets==8.1           # via sanic, synse_server (setup.py)

# The following packages are considered to be unsafe in a requirements file:
# setuptools
